.devcontainer/docker-compose-local.yaml

```diff
@@ -58,7 +58,7 @@ services:
       # NOTE: the following path contains a hash which needs to be updated whenever
       # the list of libraries installed in backend/services/experiments.py changes.
       # This is a temporary solution, see https://github.com/mozilla-ai/lumigator/pull/186
-      - LD_PRELOAD=/tmp/ray/session_latest/runtime_resources/pip/96de31139e548d3e8200d1e4b2575b77d5695312/virtualenv/lib/python3.11/site-packages/scikit_learn.libs/libgomp-d22c30c5.so.1.0.0
+      - LD_PRELOAD=/tmp/ray/session_latest/runtime_resources/pip/43b9b8b429ea9ef2b7b388371c223b0dacf13906/virtualenv/lib/python3.11/site-packages/scikit_learn.libs/libgomp-d22c30c5.so.1.0.0
```

Comment:
Note that as we updated deps this is the new hash. @veekaybee I know you have something similar in your PR right now, just a heads up so we can sync with rebase / merge

Reply:
Yup, thanks for the heads up on this. We'll really need to solve this problem at a higher-level soon :\

Reply:
We can! I'll send a PR with a fix that gets that path from the worker's ENV